docs/src/ensemble_kalman_inversion.md

```diff
@@ -1,37 +1,56 @@
 
-\\[ y = \mathcal{G}(\theta) + \eta, \\]
+```math
+y = \mathcal{G}(\theta) + \eta,
+```
+
+where ``\mathcal{G}`` denotes the forward map, ``y \in \mathbb{R}^d`` is the vector of observations
+and ``\eta  \in \mathbb{R}^d`` is additive Gaussian observational noise. Here, we take 
+``\eta \sim \mathcal{N}(0, \Gamma_y)`` from a multivariate ``d``-dimensional normal distribution
+with zero mean and covariance matrix ``\Gamma_y``.  This noise structure aims to represent the
+uncertainty, including correlated uncertainty, in the observations. Note that ``p`` is the
+size of the parameter vector ``\theta`` and ``d`` is taken to be the size of the observation
+vector ``y``.
 
-where $\mathcal{G}$ denotes the forward map, $y \in \mathbb{R}^d$ is the vector of observations and $\eta \sim \mathcal{N}(0, \Gamma_y)$ is additive Gaussian observational noise. Note that $p$ is the size of the parameter vector $\theta$ and $d$ is taken to be the size of the observation vector $y$. The EKI update equation for parameter vector $\theta^{(j)}$ of ensemble member $j$ is
+The parameter vector of the ``j``-th ensemble member at the ``n``-th iteration is ``\theta^{(j)}_n``. The EKI update equation for parameter vector ``\theta^{(j)}`` is
 
 ```math
-\theta_{n+1}^{(j)} = \theta_{n}^{(j)} - \dfrac{\Delta t_n}{J}\sum_{k=1}^J\langle \mathcal{G}(\theta_n^{(k)}) - \bar{\mathcal{G}}_n, \Gamma_y^{-1}(\mathcal{G}(\theta_n^{(j)}) - y) \rangle \theta_{n}^{(k)},
+\theta_{n+1}^{(j)} = \theta_{n}^{(j)} - \dfrac{\Delta t_n}{J}\sum_{k=1}^J \left \langle \frac{\mathcal{G}(\theta_n^{(k)}) - \bar{\mathcal{G}}_n}{\sqrt{\Gamma_y}} \, , \, \frac{\mathcal{G}(\theta_n^{(j)}) - y}{\sqrt{\Gamma_y}} \right \rangle \theta_{n}^{(k)} ,
 ```
 
-where the subscript $n=1, \dots, N_{it}$ indicates the iteration, $J$ is the number of members in the ensemble and $\bar{\mathcal{G}}_n$ is the mean value of $\mathcal{G}(\theta)$ across ensemble members,
+where the subscript ``n=1, \dots, N_{it}`` indicates the iteration, ``J`` is the number of
+members in the ensemble, ``\bar{\mathcal{G}}_n`` is the mean value of ``\mathcal{G}(\theta_n)``
+across ensemble members,
 
 ```math
-\bar{\mathcal{G}}_n = \dfrac{1}{J}\sum_{k=1}^J\mathcal{G}(\theta_n^{(k)}).
+\bar{\mathcal{G}}_n = \dfrac{1}{J}\sum_{k=1}^J\mathcal{G}(\theta_n^{(k)}) ,
 ```
 
-The EKI algorithm is considered converged when the ensemble achieves sufficient consensus/collapse in parameter space. The final estimate $\bar{\theta}_{N_{it}}$ is taken to be the ensemble mean at the final iteration,
+and angle brackets denote the Euclidean inner product. The covariance matrix ``\Gamma_y`` is
+positive-definite and, therefore, raising it to the power ``-1/2`` is meaningful. By normalizing
+with ``\Gamma_y^{-1/2}`` we render the elements inside the inner product non-dimensional.
+
```

Comment:
Remove the positive definiteness and -1/2 stuff. We can keep the mention of nondimensionalization, I think this is relevant to user understanding.